### PR TITLE
Updated refs for crypto-refresh to point to RFC9580. Formatted refere…

### PR DESCRIPTION
**Pull request by fluppe2**
…nces to sections in external RFCs to be direct hyperlinks.

This adresses #134 as well as #61.

### CODE REVIEW

File: draft-ietf-openpgp-pqc.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ The document specifies the conventions for interoperability between compliant Op
 
 ### Terminology for Multi-Algorithm Schemes
 
-The terminology in this document is oriented towards the definitions in {{?I-D.ietf-pquip-pqt-hybrid-terminology}}.
+The terminology in this document is oriented towards the definitions in [RFC9580].
```
Comment (falko-strenzke) [inline draft-ietf-openpgp-pqc.md:256]: This was a different reference.
```suggestion
The terminology in this document is oriented towards the definitions in {{?I-D.ietf-pquip-pqt-hybrid-terminology}}.
```

File: draft-ietf-openpgp-pqc.md [hunk 2]
```diff
@@ -379,13 +379,13 @@ The ML-DSA + EdDSA signature consists of independent ML-DSA and EdDSA signatures
 The OpenPGP message format allows multiple signatures of a message, i.e. the attachment of multiple signature packets.
 
 An implementation MAY sign a message with a traditional key and a PQ(/T) key from the same sender.
-This ensures backwards compatibility due to {{I-D.ietf-openpgp-crypto-refresh}}, Section 5.2.5, since a legacy implementation without PQ(/T) support can fall back on the traditional signature.
+This ensures backwards compatibility due to [[RFC9580, Section 5.2.5]](https://www.rfc-editor.org/rfc/rfc9580#name-malformed-and-unknown-signa), since a legacy implementation without PQ(/T) support can fall back on the traditional signature.
```
Comment (falko-strenzke) [inline draft-ietf-openpgp-pqc.md:382]: I am not entirely sure whether this way of rendering the reference is common practice. In the new rendering, there is no more a link to the reference section in the same document. This means that the references to RFC 9580 without a specific section point to the reference section, and the references to the RFC with a specific section point to the URL.  Does anyone know an example of an RFC which uses references in this way?

According to https://www.rfc-editor.org/styleguide/part2/ the new variant seems to be OK, as the prescription there only pertains to the rendering of the references-section, and not to how the references in the text have to look.

@TJ-91 @wussler Any opinions on this?

**Review by falko-strenzke — CHANGES_REQUESTED** [review]

File: draft-ietf-openpgp-pqc.md (hunk 2 above)
Comment (wussler) [reply draft-ietf-openpgp-pqc.md:382]: Maybe I am being dense, but do not see the problem with this reference format

**Review by wussler — APPROVED** [review]
LGTM, if applied the comment from @falko-strenzke (terminology ref)

File: draft-ietf-openpgp-pqc.md (hunk 2 above)
Comment (falko-strenzke) [reply draft-ietf-openpgp-pqc.md:382]: Yeah, I think it'll be OK. It's just that then you don't jump to the reference section by following the link, which is the only format I have seen so far in RFCs. But in any case, if our format should turn out not be OK, someone will tell us at some point and we can still revert it. In any case it is more convenient to jump to the referenced section directly.

**Review by TJ-91 — COMMENTED** [review]
Note that you can also use the section number in the URL: https://www.rfc-editor.org/rfc/rfc9580#section-5.2.5 instead of https://www.rfc-editor.org/rfc/rfc9580#name-malformed-and-unknown-signa

This might be easier to maintain and spot errors.

File: draft-ietf-openpgp-pqc.md (hunk 1 above)
Comment (fluppe2) [reply draft-ietf-openpgp-pqc.md]: Thank you for spotting this.

File: draft-ietf-openpgp-pqc.md (hunk 2 above)
Comment (fluppe2) [reply draft-ietf-openpgp-pqc.md]: Ok, seems like I can resolve this conversation.

**Comment by fluppe2** [comment]
> Note that you can also use the section number in the URL: https://www.rfc-editor.org/rfc/rfc9580#section-5.2.5 instead of https://www.rfc-editor.org/rfc/rfc9580#name-malformed-and-unknown-signa
> 
> This might be easier to maintain and spot errors.

Applied, thank you.

**Review by falko-strenzke — APPROVED** [review]

**Pull request merged**